vpnglean/CMakeLists.txt

```diff
@@ -2,6 +2,31 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+## Let's check if the glean_parser module exists.
+execute_process(
+    COMMAND ${PYTHON_EXECUTABLE} -c "import glean_parser; print(glean_parser.__version__)"
+    OUTPUT_VARIABLE GLEAN_PARSER_VERSION
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+    ERROR_QUIET
+)
+
+## Add a static library for the generated C++ code.
+add_library(vpnglean STATIC
+    ${CMAKE_CURRENT_SOURCE_DIR}/include/glean/event.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/include/glean/ping.h
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/event.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/ping.cpp
```

Comment:
This is not really generated code though. The comment on L13 needs to be clarified.